org.osate.ui/src/org/osate/ui/wizards/AbstractNewFileWizard.xtend

```diff
@@ -217,8 +217,9 @@ abstract class AbstractNewFileWizard extends Wizard implements INewWizard {
 							fieldLabel + " cannot be empty."
 						} else if (!fieldValidators.get(fieldLabel).apply(field.text)) {
 							'''The «fieldLabel» '«field.text»' is not valid.'''
-						} else if (fieldLabel == fields.keySet.head && file.exists) {
-							"'" + getFileName(fields.values.head.text) + "' already exists."
+						} else if (fieldLabel == fields.keySet.head) {
```

Comment:
Here is where the call to new validateFileName() method is inserted.  It replaces the `file.exists` check that was previously in the conditional.